### ISSUE TITLE
Can pre-commit install offline?

### ISSUE THREAD

**Issue by veyron95** [pre-commit/pre-commit#2965]
### search you tried in the issue tracker

install offline

### describe your actual problem

I want to install pre-commit offline, because my work space is offline, please tell me how to install pre-commit offline.

### pre-commit --version

none

**Comment by asottile** [comment]
yes it can. if you'd like enterprise support please set up a support contract

**Comment by veyron95** [comment]
> yes it can. if you'd like enterprise support please set up a support contract

can you give me a simple way to install pre-commit offline ??
